python/sglang/srt/managers/schedule_batch.py

```diff
@@ -452,9 +466,20 @@ def check_finished(self):
 
             for stop_str in self.sampling_params.stop_strs:
                 if stop_str in tail_str or stop_str in self.decoded_text:
-                    self.finished_reason = FINISH_MATCHED_STR(matched=stop_str)
+                    self.finished_reason = FINISH_MATCHED_REGEX(matched=stop_str)
```

Comment:
It seems `stop_str` corresponds to `FINISH_MATCHED_STR`, and `stop_regex_strs` corresponds to `FINISH_MATCHED_REGEX`